lte/gateway/c/oai/include/ngap_messages_types.h

```diff
@@ -237,6 +237,10 @@ typedef struct itti_ngap_paging_request_s {
 
 } itti_ngap_paging_request_t;
 
+typedef enum m5g_uecontextrequest {  // ue context requested supported enum
+  M5G_UEContextRequest_requested = 1,
```

Comment:
should not this be 0?

Reply:
yes, UEContextRequest_requested value should be 0 as per spec. 
but it needs to increment by '1'  to understand the amf task.

 itti_ngap_initial_ue_message_s structure is an interface between ngap and amf task. when ngap passes the initial message to amf.
   typedef struct itti_ngap_initial_ue_message_s {
                        ...
			m5g_uecontextrequest_t ue_context_request; 
			...
	}
  if ue_context_request = 0, amf task can't identify weather  ngap filled it based on UEContextRequest IE or default value.